agents/summary_agent.py

```
@@ -9,201 +9,221 @@
-                ],
-                max_tokens=800,
-                timeout=TIMEOUTS['summary_agent']
-            )
+            content = response.choices[0].message.content
+            cost = 0.0
             
-            summary_text = response.choices[0].message.content.strip()
+            # Extract cost and routing information from Router
+            if hasattr(response, '_hidden_params'):
+                cost = response._hidden_params.get('response_cost', 0.0)
+                cost_source = response._hidden_params.get('cost_source', 'openai')
```

Ruff (0.12.2)

Local variable `cost_source` is assigned to but never used

Remove assignment to unused variable `cost_source`

(F841)